web/pages/review/cards.tsx

```diff
@@ -0,0 +1,107 @@
+import { useAuth } from "react-oidc-context";
+import { useQuery } from "@tanstack/react-query";
+import {
+    reviewFidsKey,
+    reviewModesKey,
+    useSessionStorage,
+} from "@src/useStorage";
+import { getRequest } from "@src/apiRequest";
+import ProtectedRoute from "@components/ProtectedRoute";
+import Flip from "@components/routeReview/Flip";
+import { NextFlashcardGetReq } from "@src/types/NextFlashcardGetReq";
+import { NextFlashcardGetResp } from "@src/types/NextFlashcardGetResp";
+import { FlashcardFromIDResp } from "@src/types/FlashcardFromIDResp";
+import { FlashcardFromIDGet } from "@src/types/FlashcardFromIDGet";
+import Button from "@components/Button";
+/*
+This page is accessed when the user has set modes and flashcards in localStorage. 
+This page sends a request to /review/next to get the next 'packet' of
```

Comment:
For v2, `packet` might be a bit ambiguous as packet is already a term for networking stuff. Maybe not an issue, and I don't have any other ideas in mind, but worth discussing?

Reply:
Yeah, I used packet because this is kind of analogous to the usage in network terminology. I changed it to 'bunch', but feel free to replace this if you like - it's probably bikeshedding to discuss it any further.